runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -155,42 +158,54 @@ public void onConfigChanged(ConfigChanged event)
 				frame.setResizable(!config.lockWindowSize());
 			}
 
-			if (!event.getKey().equals("gameSize"))
+			if (client == null)
 			{
 				return;
 			}
 
-			if (client == null)
+			if (event.getKey().equals("gameSize"))
```

Comment:
Any reason why was this inverted to giant if condition? The maximized configuration can be moved before this if and you can leave the original early return if as it was before.